src/theory/relevance_manager.h

```diff
@@ -73,7 +79,7 @@ class RelevanceManager
   typedef context::CDList<Node> NodeList;
 
  public:
-  RelevanceManager(context::UserContext* userContext, Valuation val);
+  RelevanceManager(Env& env, Valuation val);
```

Comment:
Should we make `RelevanceManager` an `EnvObj`?

Reply:
Actually this is unecessary for now, I'll revert this.